src/main/java/org/logstash/plugins/inputs/http/HttpInitializer.java

```diff
@@ -0,0 +1,46 @@
+package org.logstash.plugins.inputs.http;
+
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.http.HttpContentDecompressor;
+import io.netty.handler.codec.http.HttpObjectAggregator;
+import io.netty.handler.codec.http.HttpServerCodec;
+import io.netty.handler.ssl.SslHandler;
+import org.logstash.plugins.inputs.http.util.SslSimpleBuilder;
+
+import java.util.concurrent.ThreadPoolExecutor;
+
+/**
+ * Created by joaoduarte on 11/10/2017.
+ */
+public class HttpInitializer extends ChannelInitializer<SocketChannel> {
+    private final IMessageHandler messageHandler;
+    private SslSimpleBuilder sslSimpleBuilder;
+    private int maxContentLength = 100 * 1024 * 1024;
```

Comment:
can you pass this via the constructor , it looks like you already have it in scope for NettyHttpServer